src/inmanta/export.py

```diff
@@ -480,6 +480,14 @@ def upload_file(self, content: Union[str, bytes]) -> str:
 
         return hash_id
 
+    def get_environment_id(self) -> str:
+        env = str(Config.get("config", "environment", None))
```

Comment:
Use cfg_env which is acutally defined in this file

Reply:
resolved